src/pages/City/CreateCity/CreateCity.tsx

```diff
@@ -273,6 +274,10 @@ const CreateCity = () => {
     CreateCity(newCity, regionFollower.id);
   };
 
+  const getOnlyNums = (text: string) => {
```

Comment:
I believe it should be better to move that function in some common service and reuse it
also think about naming, for me it's not clear what it should get(but I think it should be numbers)